### DISCUSSION TITLE
The problem that the half-life does not decrease uniformly only by adjusting Vmax

### DISCUSSION THREAD

**Discussion by gapa84** [Open-Systems-Pharmacology/Forum#994] [Questions & Problems]
Hello.
I am getting a lot of help here.
The half-life of the drug being simulated is getting too long. Adjusting Vmax decreases the half-life and increases it non-constantly. When the Vmax value is increased, the half-life decreases for the first few times and then increases abruptly. What other variables besides Vmax should be adjusted to reduce the half-life?

**Comment by TWendl** [comment]
Hi!
Half-life may also be influenced by absorption and redistribution, so, you could adjust parameters related to these processes. I would recomment to display the "fraction absorbed into mucosa" and the organ intracellular concentrations in your simulations in order to evaluate their influence.

**Reply by gapa84** [reply]
Where can I check the value you mentioned?
thank you.

**Reply by TWendl** [reply]
Open your simulation, click "define settings and run", scroll to "Lumen", select "Fraction of oral drug mass absorbed".
Please have a look in the manual for  further information !

![fractionOfOralDrugMassAbsorbed](https://user-images.githubusercontent.com/26460649/138421852-4dae9588-e90b-431a-9cde-08b7d0014499.PNG)

**Reply by gapa84** [reply]
![image](https://user-images.githubusercontent.com/92776578/138539985-c9747549-f9eb-4ece-ac45-3fe7a00a9178.png)

Thank you very much. I'll let you know the value you mentioned.

**Comment by StephanSchaller** [comment]
@gapa84 , try simulating for a longer time. Terminal half-life should be close to zero of you reduce al clearance processes (i.e. set vmax, kcat, ... tp zero).
As mentioned by @TWendl , the decline you see in the beginning of drug PK is also gouerned by absorption and distribution processes (e.g. permeability and partitioning values)
